src/styles/functions.scss

```diff
@@ -0,0 +1,23 @@
+
+$week-days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
```

Comment:
According to the code style rules, the map keys and values should be aligned, which means there should be a single space before and after each colon. Please adjust the alignment for readability.